nasa/earthdata-search
# EDSC-3517: Preventing decoding of facet values

# Overview

### What is the feature?

Fixing a bug where encoded facet values led to incorrectly applied facets

### What is the Solution?

Do not decode the values provided by CMR

### What areas of the application does this impact?

search/facets

# Testing

### Reproduction steps

- **Environment for testing:** PROD
- **Collection to test with:** n/a

1. Visit `/search?fdc=Level-1%2Band%2BAtmosphere%2BArchive%2B%2526%2BDistribution%2BSystem%2B%2528LAADS%2529`
2. Confirm the correct facet is applied
3. Refresh the page
4. Confirm the correct facet is still applied

# Checklist
